src/databricks/labs/lsql/dashboards.py

```diff
@@ -23,18 +22,33 @@
     Dashboard,
     Dataset,
     Field,
+    Json,
     Layout,
     NamedQuery,
     Page,
     Position,
     Query,
     Widget,
+    WidgetSpec,
 )
 
+_MAXIMUM_DASHBOARD_WIDTH = 6
 T = TypeVar("T")
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class MarkdownSpec(WidgetSpec):
+    """A dummy spec for markdown files."""
```

Comment:
Do we need it?..

Reply:
No. I overengineerd that, thanks for spotting it